package.json

```diff
@@ -11,8 +11,10 @@
     "bootstrap": "npx gulp bootstrap",
     "develop": "npx gulp develop",
     "lint": "npx gulp lintAll",
+    "lint:md": "npx gulp prettify",
```

Comment:
we should add a filter for only markdown here. We will use prettify for js etc in the future as well.

Reply:
Unclear on the API here. Would it be `gulp prettify lint --md` and `gulp prettify format --md`?

Reply:
`lint` would be the default subcommand, so it could be omitted. Alternatively, they could both be flags (`--format --md`). Also, any preference between `--fix` and `--format`?

Reply:
maybe simply go with `lint:prettify` and don't add a file format filter